data.sql

```diff
@@ -1,5 +1,8 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals ( name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES 
+('Agumon','2020-02-03',0,TRUE,10.23),
+('Gabumon','2018-11-15',2,TRUE,8.0),
+('Pikachu','2021-01-07',1,FALSE,15.04),
+('Devimon','2017,05,12',1,FALSE,15.04);
+
```

Comment:
- [ ] The weight for the animal with the name `Devimon` should be `11.0` , the `escape_attempts` is also `'5'` and the `neutered` is on `'true'`. Kindly, correct your queries.

Reply:
Thanks for the observation, I have updated the values :wink: